src/Adapter/Attribute/Repository/AttributeRepository.php

```diff
@@ -117,26 +117,12 @@ public function getProductAttributesIds(ProductId $productId): array
     public function getAttributesInfoByCombinationIds(array $combinationIds, LanguageId $langId): array
     {
         $attributeCombinationAssociations = $this->getAttributeCombinationAssociations($combinationIds);
+        $attributesInfoByAttributeId = $this->getAttributesInfoByAttributeId($attributeCombinationAssociations, $langId);
```

Comment:
as this all purely a little cleaning? you didn't change any of the structure right? (because these methods are already used in couple places I believe)

Reply:
Logic is the same, I've just extract some code into privates